src/common/components/elements/Avatar.tsx

```diff
@@ -10,10 +10,23 @@ import { useWallet } from '@solana/wallet-adapter-react';
 
 export const AvatarIcons = ({ creators }: { creators: { address: string }[] }) => {
   return (
-    <div className="relative">
-      {creators.map(({ address }, i) => (
-        <AvatarIcon address={address} index={i} key={address} />
+    // wrap the avatars in a container with a small, colored background
+    <div className={`inline-flex items-center rounded-full bg-gray-600 bg-opacity-70 p-1`}>
+      {creators.slice(0, 4).map(({ address }, i) => (
+        <div key={address} className={`h-6 w-6 ${i > 0 ? '-ml-3' : ''}`}>
```

Comment:
Check out classnames package for doing conditional classnames.

https://www.npmjs.com/package/classnames

Already included in the project.